windows/rngallery/App.cpp

```diff
@@ -41,6 +41,7 @@ App::App() noexcept
 
     PackageProviders().Append(make<ReactPackageProvider>()); // Includes all modules in this project
     PackageProviders().Append(winrt::CheckboxWindows::ReactPackageProvider());
+    PackageProviders().Append(winrt::DateTimePicker::ReactPackageProvider());
```

Comment:
Please add a TODO to remove these lines of code once issue [https://github.com/microsoft/react-native-gallery/issues/9](https://github.com/microsoft/react-native-gallery/issues/9) is fixed.